dev/PushNotifications/PushNotificationsLongRunningTask/ForegroundSinkManager.h

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include <unordered_map>
+#include <unordered_set>
+#include <algorithm>
+#include <vector>
+
+class ForegroundSinkManager
+{
+public:
+    ForegroundSinkManager() = default;
+
+    void AddSink(std::wstring processName, IWpnForegroundSink* sink);
+
+    void Remove(std::wstring processName);
+
+    bool InvokeForegroundHandlers(std::wstring processName, byte* payload, ULONG payloadSize);
```

Comment:
Should this be a `winrt::com_array<>` or a `gsl::span` ? Don't use raw pointers unless you have to.